qupath-gui-fx/src/main/java/qupath/lib/gui/actions/ActionTools.java

```diff
@@ -454,12 +451,27 @@ private static void parseConfig(Action action, ActionConfig annotation, Localize
 				localizedResourceManager.registerProperty(action.textProperty(), bundle, key);
 			else
 				action.setText(QuPathResources.getString(bundle, key));
+
 			String descriptionKey = key + ".description";
 			if (QuPathResources.hasString(bundle, descriptionKey)) {
-				if (annotation.bindLocale())
-					localizedResourceManager.registerProperty(action.longTextProperty(), bundle, descriptionKey);
-				else
-					action.setLongText(QuPathResources.getString(bundle, descriptionKey));
+				if (annotation.bindLocale()) {
+					StringProperty resourceLongTextProperty = new SimpleStringProperty();
+					resourceLongTextProperty.addListener((p, o, n) -> {
+						String longText = n;
+						if (action.getAccelerator() != null) {
+							longText = "(" + action.getAccelerator() + ") " + longText;
+						}
+						action.setLongText(longText);
+					});
+					localizedResourceManager.registerProperty(resourceLongTextProperty, bundle, descriptionKey);
+				}
+				else {
+					String longText = QuPathResources.getString(bundle, descriptionKey);
+					if (action.getAccelerator() != null) {
```

Comment:
Would it be 'cleaner' to get the long text for an `Action` in a method, to avoid the duplication?

Reply:
Yes